Sources/Mantis/Helpers/TransformStack.swift

```diff
@@ -0,0 +1,59 @@
+import UIKit
+
+
+class TransformStack: NSObject {
+    
+    public static var shared: TransformStack = TransformStack()
+    
+    weak var transformDelegate : TransformDelegate!
+    
+    public var transformAdjustmentsStack : Array = [TransformRecord]()
+    
+    public var isRotated : Bool! = false
+    
+    public var bottom : Int = 0
+    public var top : Int = 0
+    
+    public var transformStackBottom : Int = 0
+    public var transformStackTop : Int = 0
+        
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override init() {
+        super.init()
+        
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(self.undoStatusChanged(notification:)),
+            name: .NSUndoManagerCheckpoint,
+            object: nil)
+    }
+    
+    public func pushTransformRecord(_ record: TransformRecord)
+    {
+        if transformAdjustmentsStack.count > top {
+            transformAdjustmentsStack.remove(at: top)
+        }
+        transformAdjustmentsStack.insert(record, at: top)
+        top += 1
+    }
+
+    public func popTransformStack()
+    {
```

Comment:
This `{` needs to be moved to the same line of `public func popTransformStack()`

Reply:
changes made.